venues/DBLP.org/python/dblp_data.py

```diff
@@ -0,0 +1,3 @@
+BASE = 'DBLP.org'
+GROUP = BASE + '/upload'
```

Comment:
After looking through this, I'm not sure that we need this "DBLP.org/upload" group after all. Does it serve some purpose that the DBLP.org group can't?